actors/miner/src/lib.rs

```diff
@@ -3813,15 +3819,16 @@ fn extend_non_simple_qap_sector(
     let mut new_sector = sector.clone();
     // Remove "spent" deal weights for non simple_qa_power sectors with deal weight > 0
     let new_deal_weight = (&sector.deal_weight * (sector.expiration - curr_epoch))
-        .div_floor(&BigInt::from(sector.expiration - sector.activation));
+        .div_floor(&BigInt::from(sector.expiration - sector.power_base_epoch));
```

Comment:
checking for understanding -- this will only ever be different from activation in the case of a Prove Replica Update before the shark upgrade since non-simple sectors never become simple and non simple extensions will never modify power base epoch?

Reply:
My updated understanding is they will update the power_base_epoch